browser-test/login.test.js

```diff
@@ -47,5 +48,11 @@ describe("Selenium tests for <Login />", () => {
     await driver.wait(until.elementIsVisible(successToastDiv));
     await driver.wait(until.urlContains("status"), 5000);
     expect(await successToastDiv.getText()).toEqual("Login successful");
+
+    const activeSessionTr = await getElementByCss(
+      driver,
+      "table tr.active-session",
+    );
+    await driver.wait(until.elementIsVisible(activeSessionTr));
```

Comment:
unfortunately this doesn't fail for me without the fix, I was not able to replicate this issue in automated tests, only with manual testing :(

Reply:
Yes, I also tried but sessions are showing in the browser test.